tests/core/filtering/test_contract_topic_filters.py

```diff
@@ -135,6 +102,30 @@ def array_values(draw):
     return (matching, non_matching)
 
 
+# --- sync --- #
+
+
+@pytest.fixture(
+    scope="module",
+    params=[True, False],
+    ids=["local_filter_middleware", "node_based_filter"],
+)
+def w3(request):
+    return _w3_fixture_logic(request)
+
+
+@pytest.fixture(scope="module")
+def Emitter(w3, EMITTER):
+    return w3.eth.contract(**EMITTER)
+
+
+@pytest.fixture(scope="module")
+def emitter(w3, Emitter, wait_for_transaction, wait_for_block, address_conversion_func):
+    return _emitter_fixture_logic(
```

Comment:
nit: I'm not sure I see the value in having a function just pass on it's args to another function. Can we move the logic back here and avoid some indirection? Alternatively, if this is used in more places than it looks like it is in this PR, can we pull the fixture up a few levels?

Reply:
The other use is in the local `conftest.py`, which came through in the previous PR. The fixture is redefined here due to scope. I had pulled the logic of the longer fixtures out to `utils.py` to avoid duplication. It is only used twice though, so I'm happy to move the logic back if you would prefer.

Reply:
And I'll aim to have these sorts of related changes come through in a single PR in the future.

Reply:
Ah, right. I forgot about those warnings. I don't have a strong preference either way. I think the real fix is to figure out why we can't use this fixture on the higher levels without warnings. I just added another bullet point to issue #2460 to look into this.

Reply:
In this case, it's hypothesis that has issues with the fixture scope. Not sure about other cases.